docs/reference/commandline/dockerd.md

```diff
@@ -977,3 +977,39 @@ has been provided in flags and `cluster-advertise` not, `cluster-advertise`
 can be added in the configuration file without accompanied by `--cluster-store`
 Configuration reload will log a warning message if it detects a change in
 previously configured cluster configurations.
+
+
+## Running multiple daemons
+
+From docker 1.9.1 it is possible to run two Docker daemons on a single host.
```

Comment:
Agreed that the version number is not needed here (we generally keep this out of the documentation, as that's what the change log is for). Perhaps something like;

```Markdown
This section describes how to run multiple Docker daemons on a single host. To
run multiple daemons, you must configure each daemon so that it does not
conflict with other daemons on the same host. You can set these options either
by providing them as flags, or by using a [daemon configuration file](#daemon-configuration-file).

The following daemon options must be configured for each daemon:
```